sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -175,19 +187,71 @@ def test_check_estimator():
 
     # doesn't error on actual estimator
     check_estimator(AdaBoostClassifier)
+    check_estimator(AdaBoostClassifier())
     check_estimator(MultiTaskElasticNet)
+    check_estimator(MultiTaskElasticNet())
+
+
+def _bad_fit(self, X, y):
+    raise ValueError("You shouldn't have called this")
+
+
+def test_check_estimator_clones():
+    # check that check_estimator doesn't modify the estimator it receives
```

Comment:
Aren't you doing more than that in the test? Maybe the comment above should be updated?